pkg/scheduler/apis/config/v1beta3/default_plugins_test.go

```diff
@@ -454,6 +454,37 @@ func TestMergePlugins(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "disabled MultiPoint plugin in default set",
+			defaultPlugins: &v1beta3.Plugins{
+				MultiPoint: v1beta3.PluginSet{
+					Enabled: []v1beta3.Plugin{
+						{Name: "DefaultPlugin"},
+					},
+					Disabled: []v1beta3.Plugin{
```

Comment:
the more important case is where you disable an extension, not the whole plugin.

Reply:
That already has its own test case above https://github.com/kubernetes/kubernetes/pull/105611/files#diff-392546fd01ec251b85f84db8e534ad5d22aa4e58de40b629417d0af0f90916b9R390-R432

Reply:
that test doesn't cover the case where an extension point is disabled in the default set.

Reply:
Ah, I see, misread that. Added